### PR TITLE
posix: missing closing parenthesis

### PR DESCRIPTION
**Pull request by OlegHahm**
Apparently this was never built.

### CODE REVIEW

**Comment by gebart** [comment]
Does it work at all?

File: sys/posix/sockets/posix_sockets.c [hunk 1]
```diff
@@ -795,7 +795,7 @@ ssize_t recvfrom(int socket, void *restrict buffer, size_t length, int flags,
         case SOCK_RAW:
             /* TODO: apply configured timeout */
             if ((res = sock_ip_recv(&s->sock->raw, buffer, length, SOCK_NO_TIMEOUT,
```
Comment (A-Paul) [inline sys/posix/sockets/posix_sockets.c]: How about something like this? :)
```
            res = sock_ip_recv(&s->sock->raw, buffer, length, SOCK_NO_TIMEOUT,
                    (sock_ip_ep_t *)&ep);
            if (res < 0) {
                errno = -res;
                res = -1;
            }
```

**Comment by OlegHahm** [comment]
@gebart, without this PR certainly not. This PR increases the chances. ;)

File: sys/posix/sockets/posix_sockets.c (hunk 1 above)
Comment (OlegHahm) [inline sys/posix/sockets/posix_sockets.c]: Personally, I don't see any benefit, but feel free to provide a PR against my PR, but then please update all following cases accordingly for consistency.

**Pull request merged**